test/build.sh

```diff
@@ -42,9 +42,10 @@ trap cleanup EXIT
 
 set -x
 latest=$(readlink ./releases/cri-cni-containerd.tar.gz)
-cp releases/${latest} ${BUILDDIR}/cri-containerd.tar.gz
-cp releases/${latest}.sha256sum ${BUILDDIR}/cri-containerd.tar.gz.sha256
+tarball=$(echo ${latest} | sed -e 's/cri-containerd-cni/containerd-cni/g' | sed -e 's/-linux-amd64/.linux-amd64/g')
```

Comment:
Is it `s/cri-cni-containerd/containerd-cni/g` ?

Reply:
https://github.com/containerd/containerd/blob/master/Makefile#L69

This is right. A few different variations in use today